apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -258,25 +256,6 @@ impl Plugin for Telemetry {
             .boxed()
     }
 
-    fn query_planner_service(
-        &self,
-        service: query_planner::BoxService,
-    ) -> query_planner::BoxService {
-        ServiceBuilder::new()
-            .instrument(move |req: &QueryPlannerRequest| {
-                let query = req.query.clone();
-                let operation_name = req.operation_name.clone().unwrap_or_default();
-
-                info_span!("query_planning",
-                    graphql.document = query.as_str(),
-                    graphql.operation.name = operation_name.as_str(),
-                    "otel.kind" = %SpanKind::Internal
-                )
-            })
```

Comment:
todo: put it somewhere else (this will prove non trivial because of clone / send / sync requirements practically everywhere)

Reply:
good to go!